README.md

```diff
@@ -1,13 +1,20 @@
 # vitasdk code samples
 
-Every directory contains a single sample. To build all samples, add the toolchain `bin/` directory to your `$PATH`, then run `./build.sh`. To clean all samples, run `./clean.sh`. To build a single sample, `cd` to its directory and type `make`.
+## Prerequists
 
-Always use the `*_debug.bat` (e.g. `run_homebrew_unity_debug.bat`) launcher to get debug output from Vita.
+In order to build a vita sample, you need to add the toolchain `bin/` directory to your `$PATH`.
 
-## List of samples
+## Building
 
-* `hello_world`: A minimal Makefile-based C project
-* `hello_cpp_world`: A minimal Makefile-based C++ project
```

Comment:
Why did you delete this? It would be nice to have short sample descriptions here, with optionally a longer README.md in the sample directory.